.github/workflows/spm.yml

```diff
@@ -0,0 +1,15 @@
+name: spm
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  swift-build-run:
+    runs-on: macOS-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
+      run: swift build
+    - name: Run
+      run: swift test
```

Comment:
As I mentioned before OCMock uses Travis CI. Please do not add another CI system.

Reply:
Does this PR become mergeable by moving the CI to Travis or are additional changes needed?

Reply:
As I wrote last year, I'm okay with not having any tests for this. So, there isn't even a need to move this to Travis.

Regarding the status of the PR, there's still the open question around what should go into the `.modulemap` file. The discussion seemed to have petered out, which is why I asked for clarification earlier today.

Reply:
If it can be merged, my preference would be to add the tests, to make it easier to maintain going forward. I'm fine to do that migration if you're ok with SPM tests in `.travis.yml`.  The [GHA tests run in a minute or two](https://github.com/firebase/ocmock/actions), so it should be minimal impact.

On the `.modulemap` discussion, since the current state has worked for us and others using the fork, my recommendation would be to keep it as is and address any shortfalls in another PR.

Reply:
Okay, that means the `.modulemap` is good to go. If you want to add a test on CI, please don't add it directly to the Travis file. That only invokes `make ci`. So, the actual test should go into the makefile.